### PR TITLE
Updated readme with sdk version migration instructions

### PR DESCRIPTION
**Pull request by travega**

### CODE REVIEW

**Comment by travega** [comment]
Hey @DaleCantwell can you have a look at these SDK migrations docs updates and let me know if they need work? 👍

File: README.md [hunk 1]
```diff
@@ -175,3 +175,38 @@ Detailed documentation and getting started guides for:
...
+
+This will ensure you will get the most current version of the SDK when as it is released.
+
+Our Android SDK versions come bundled with some 3rd party dependencies. To ensure you don't run into any version conflicts here is a list of the dependency versions we ship with our latest SDK version:
+
+<table>
+  <tr><th colspan="2" style="text-align: center;">Android</th></tr>
+  <tr><td style="text-align: center; font-weight: bold; width: 50%;">Library</td><td style="text-align: center; font-weight: bold; width: 50%;">Version</td></tr>
+  <tr><td style="width: 50%;">com.google.code.gson</td><td style="width: 50%;">2.5</td></tr>
+  <tr><td style="width: 50%;">com.squareup</td><td style="width: 50%;">1.3.7</td></tr>
```
Comment (ronocod) [inline README.md]: This should read `com.squareup.otto`

**Comment by travega** [comment]
Good spot 👍 I'll push that change now

**Comment by ronocod** [comment]
🚢

**Comment by travega** [comment]
👍

**Pull request merged**